src/AzSK.ADO/Framework/Configurations/SVT/ADO/ADO.Organization.json

```diff
@@ -672,6 +672,25 @@
         "AutomatedFix"
       ],
       "Enabled": true
-  }
+    },
+    {
+      "ControlID": "ADO_Organization_DP_Disable_Creation_Of_Classic_Pipeline",
+      "Description": "Disable the creation of classic build and classic release pipelines.",
+      "Id": "Organization500",
+      "ControlSeverity": "Medium",
```

Comment:
How we decided the severity?

Reply:
The same control is implemented in AzTS framework and there the severity is medium.
So I kept it as medium.